### PR TITLE
Upgrade Unrealircd and move Anope to separate container

### PR DESCRIPTION
**Pull request by p4block**
More work here in the future. The Unrealircd container is not the absolute latest version, but it's much better than before.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -372,12 +368,35 @@ services:
       - "8067:8067"
       - "7070:7070"
       - "8167:8167"
+    expose:
+      - 7000
```
Comment (Brutus5000) [inline docker-compose.yml:363]: Add a comment that it's open for anope to connect

File: docker-compose.yml [hunk 2]
```diff
@@ -341,29 +341,25 @@ services:
...
-      - ./config/faf-unrealircd/services.conf:/home/unreal/unrealircd/services/conf/services.conf
-      - ./config/faf-unrealircd/unrealircd.conf:/home/unreal/unrealircd/conf/unrealircd.conf
-      - ./config/faf-unrealircd/motd.txt:/home/unreal/unrealircd/conf/ircd.motd
+      - ./config/faf-ircd/ssl/server.cert.pem:/data/unrealircd/conf/tls/server.cert.pem
+      - ./config/faf-ircd/ssl/server.key.pem:/data/unrealircd/conf/tls/server.key.pem
+      - ./config/faf-ircd/unrealircd.conf:/data/unrealircd/conf/unrealircd.conf
+      - ./config/faf-ircd/motd.txt:/data/unrealircd/conf/ircd.motd
+      - ./data/faf-ircd/:/data/unrealircd/data/
     ports:
       - "6667:6667"
```
Comment (Brutus5000) [inline docker-compose.yml]: Reduce to port 6667 only?

File: docker-compose.yml [hunk 3]
```diff
@@ -372,12 +368,35 @@ services:
...
+  faf-anope:
+    container_name: faf-anope
+    image: anope/anope
+    restart: unless-stopped
+    networks:
+      - faf
+    depends_on:
+      - faf-ircd
+    volumes:
+      - ./config/faf-anope/services.conf:/anope/conf/services.conf
```
Comment (Brutus5000) [inline docker-compose.yml:383]: I'd always prefer to mount the whole folder if possible

File: docker-compose.yml [hunk 4]
```diff
@@ -341,29 +341,25 @@ services:
   #
   # IRC for FAF chat.
   #
-  faf-unrealircd:
-    container_name: faf-unrealircd
-    image: faforever/faf-unrealircd:v1.2
-    user: ${FAF_UNREALIRCD_USER}
+  faf-ircd:
+    container_name: faf-ircd
+    image: bbriggs/unrealircd
```
Comment (Brutus5000) [inline docker-compose.yml]: Reminder: we need to fork and update to latest version

**Review by Brutus5000 — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 3 above)
Comment (p4block) [reply docker-compose.yml:383]: That folder contains a dozen of stock configuration files we want to keep upstream, so file is the way

File: docker-compose.yml (hunk 2 above)
Comment (p4block) [reply docker-compose.yml]: I do agree with this, but will need also a configuration file slimming

**Review by Brutus5000 — APPROVED** [review]

**Pull request merged**